### PR TITLE
fix(web-app): update side nav z-index to fix tab bg overlay with scrollbar

### LINKED ISSUE #1078: Page tabs container is offset by one scrollbar-width
**Description**

When viewing a page with a page tabs component on it, if the page's content has a scrollbar, it pushes the tabs container over to the left by the width of a scrollbar.

image

image

Image from iOS

**Steps to reproduce**

Example page: https://next.carbondesignsystem.com/elements/accessibility/overview
Windows - should be visible without extra config
Mac - Go into "settings -> general -> show scroll bars" and set it to "always".

Tried both Chrome and Edge on Mac, and Edge on Windows.

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #1078

#### Changelog

**Changed**

- update side nav z-index so it sits above page tabs

#### Testing / reviewing

http://localhost:3000/guidelines/accessibility/overview

Test with scrollbars on and off and make sure pages with tabs render correctly
Screen Shot 2022-08-16 at 4 23 07 PM

### CODE REVIEW

**Review by francinelucca — APPROVED** [review]
that fixed it for me!
(Found another bug but opened separate ticket: #1147)

File: services/web-app/layouts/layout/layout.module.scss [hunk 1]
```diff
@@ -11,6 +11,8 @@
 @use '@carbon/react/scss/motion' as motion;
 @use '@carbon/react/scss/type' as type;
 @use '@carbon/react/scss/utilities/convert' as convert;
+@use '../../styles/variables' as variables;
+@use 'sass:map';
```
Comment (jdharvey-ibm) [inline services/web-app/layouts/layout/layout.module.scss:15]: minor: move built-in modules to top of the `@use` list.

**Review by jdharvey-ibm — APPROVED** [review]
local test looked good to me!

File: services/web-app/layouts/layout/layout.module.scss (hunk 1 above)
Comment (alisonjoseph) [reply services/web-app/layouts/layout/layout.module.scss]: done

**Pull request merged**